packages/slate/test/changes/at-current-range/insert-block/block-end-empty.js

```diff
@@ -0,0 +1,30 @@
+/** @jsx h */
+
+import h from '../../../helpers/h'
+
+export default function (change) {
+  change.insertBlock('quote')
+}
+
+export const input = (
+  <value>
+    <document>
+      <paragraph>
+        <anchor />word
+      </paragraph>
+      <paragraph>
+        <focus />
+      </paragraph>
+    </document>
+  </value>
+)
+
+export const output = (
+  <value>
+    <document>
+      <quote>
+        <cursor />
```

Comment:
This test is failing. I end up with two blocks rather than one. Am I doing something wrong here?